src/main/java/uk/gov/companieshouse/insolvency/delta/InsolvencyDeltaConsumerApplication.java

```diff
@@ -6,9 +6,10 @@
 @SpringBootApplication
 public class InsolvencyDeltaConsumerApplication {
 
-    public static final String NAMESPACE = "insolvency-delta-consumer";
+    public static final String APPLICATION_NAMESPACE = "insolvency-delta-consumer";
```

Comment:
This is typically defined as APPLICATION_NAME_SPACE, unsure if there's an impact